manager/controllers/package_systems.go

```diff
@@ -39,11 +39,11 @@ func packagesByNameQuery(pkgName string) *gorm.DB {
 		Where("pn.name = ?", pkgName)
 }
 
-func packageSystemsQuery(acc int, nameIDs []int) *gorm.DB {
+func packageSystemsQuery(acc int, packageIDs []int) *gorm.DB {
 	query := database.SystemPackages(database.Db, acc).
 		Select(PackageSystemsSelect).
 		Where("sp.stale = false").
-		Where("spkg.name_id in (?)", nameIDs)
+		Where("spkg.package_id in (?)", packageIDs)
```

Comment:
Do we have test for it? Why we didn't catch it there?

Reply:
Because name_ids were the same as package_id

Reply:
I remember exactly same  bug in old project where we didn't catch switched username & password fields because we used same value for both in tests :-) .